src/Services/ReorderElements.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace DNADesign\Elemental\Services;
+
+use InvalidArgumentException;
+use SilverStripe\Core\Convert;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\ORM\Queries\SQLUpdate;
+use DNADesign\Elemental\Models\BaseElement;
+
+class ReorderElements
+{
+    /**
+     * @var BaseElement $block
+     */
+    private $element;
+
+    public function __construct(BaseElement $element)
+    {
+        if (!is_subclass_of($element, BaseElement::class)) {
+            throw new InvalidArgumentException(sprintf(
+                'Invalid ' . BaseElement::class . ' passed to ' . __CLASS__
```

Comment:
In the new mutation you're throwing an exception which contains the passed class name, do you want to use the same thing here? I.e. `get_class($element) . ' is not an instance of BaseElement` or whatever?